python/foglamp/plugins/south/cc2650async/cc2650async.py

```diff
@@ -0,0 +1,354 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Module for Sensortag CC2650 'async' type plugin """
+
+import copy
+import datetime
+import uuid
+import json
+import asyncio
+from foglamp.plugins.south.common.sensortag_cc2650 import *
```

Comment:
add a line break; when you are doing `import *`; I assume you are using `__all___ = [...]` in common.sensortag_cc2650 module